extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -300,7 +300,8 @@ export async function getCompletionItemsFromSpecs(
 					command,
 					command.detail,
 					command.documentation,
-					vscode.TerminalCompletionItemKind.Method
+					// If the label is only dots (e.g. '..'), treat as folder, else as method
+					/^\.{2,}$/.test(labelWithoutExtension) ? vscode.TerminalCompletionItemKind.Folder : vscode.TerminalCompletionItemKind.Method
```

Comment:
This looks wrong, anything coming from specs or builtins should be a method. I think you're trying to handle this case here:

![image](https://github.com/user-attachments/assets/cacc4b91-d7fd-4772-a878-4f7010024d71)

But that's wrong since those are actually aliases (or builtins?), and while they do resolve to folders in the end, it should stay as an alias not a folder completion type.